tests/acceptance/bootstrap/FeatureContext.php

```diff
@@ -165,9 +165,16 @@ public function theSprintDoesNotExist($title)
 		}
 	}
 
-	private function getPhabricatorProjectFromTitle($title)
+	private function getOrCreatePhabricatorProjectFromTitle($title)
 	{
-		return App::make('phabricator')->queryProjectByTitle($title);
+		$phabricator =  App::make('phabricator');
+		$project = $phabricator->queryProjectByTitle($title);
+
+		if (!$project) {
```

Comment:
Curly bracket should be on the next line.

Also this would be even more compact but I don't object to the current solution:

    $phabricator = App::make('phabricator');
    return $phabricator->queryProjectByTitle($title) ?: $phabricator->createProject($title, []);